tools/README.md

```diff
@@ -0,0 +1,4 @@
+# Arcus - Tools
+Arcus provides some tooling to streamline the governance & setup of new projects.
+
+- **GitHub Repo Configurator** - Configures a GitHub repository according to a defined repository manifest.
```

Comment:
Would make this a link to the README.md of that project.

Reply:
Good suggestion!